app/scripts/controllers/transactions/index.js

```diff
@@ -155,6 +160,14 @@ export default class TransactionController extends EventEmitter {
     this.messagingSystem = opts.messenger;
     this._hasCompletedOnboarding = opts.hasCompletedOnboarding;
 
+    const { hooks } = opts ?? {};
+    this._afterSign = (...args) => hooks?.afterSign?.(...args) ?? true;
```

Comment:
Very minor, but maybe slightly cleaner to not wrap the ternaries in functions. Such as:

```
this._afterSign = hooks?.afterSign ?? () => true;
```

Reply:
It was complaining about the usage of the nullish coalescing operator so I removed the wrapped function, and end up with:  `hooks?.afterSign ? hooks.afterSign : () => true`.